### ISSUE TITLE
Encountering SSL Error While Starting Flask Server

### ISSUE THREAD

**Issue by rsgrouptime** [xlang-ai/OpenAgents#134]
Hello, I'm currently configuring the backend code on a Linux machine. I've followed the steps outlined in the README.md file within the backend folder to install the necessary packages. However, upon initiating the Flask service with the command 'flask run -p 8000', the service starts but throws an SSL error when attempting to download plugins.

I'm curious if others have experienced similar issues when starting the server. Additionally, when I manually check the plugin URL mentioned in the error snapshot via a browser, the site appears untrusted and the URL is marked as unsecured.

Below, I've provided the specific error message received while starting the Flask server. Any insights or assistance in resolving this SSL error would be greatly appreciated."

Error: Failed to make the request MAPS: HTTPSConnectionPool(host='
www.smoothplugins.com'
, port=443): Max retries exceeded with **url:** /img/plugin_logos2/maps1.png (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)')))
Error: Failed to make the request NBA_STATS: 404 Client Error: Not Found for **url**:
https://nba-gpt-prod.onrender.com/logo.png
Error: Failed to make the request DREAMINTERPRETER: HTTPSConnectionPool(host='dreamplugin.bgnetmobile.com', port=443): Max retries exceeded with **url**: /.well-known/logo.png (Caused by SSLError(SSLCertVerificationError(1, "[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: Hostname mismatch, certificate is not valid for 'dreamplugin.bgnetmobile.com'. (_ssl.c:1007)")))

**Comment by Timothyxxx** [comment]
Thanks for the question, I think it is due to the fact that these API providers no longer provide APIs as we used before.
You can simply disable these Plugins [here](https://github.com/xlang-ai/OpenAgents/blob/main/real_agents/plugins_agent/plugins/plugin_names.py) if you don't want to see these error messages.

Best,
Tianbao

**Comment by rsgrouptime** [comment]
Sure. Thank you for your response. Let me keep the required plugins and execute the same. I'll confirm you once done.  

Regards,
Ravi Sarpal

**Comment by rsgrouptime** [comment]
Hi. I have disabled the unwanted plugins and can start the backend service.

"I'm currently configuring the frontend code and have installed the necessary Node.js and npm packages. Although I can initiate the application, I've encountered an issue regarding the 'AbortController', preventing the download of 'Inter' from Google Fonts. I'm resolving this issue and will update you if there are any further internal complications.
<img width="632" alt="frontend_issue" src="https://github.com/xlang-ai/OpenAgents/assets/168621986/a9ccb71d-e09c-4073-bf95-da65d63714ef">
